thirdparty/README.md

```diff
@@ -31,13 +31,12 @@ following folders removed:
 
 ## autodiff
 
-A copy of [autodiff](https://github.com/autodiff/autodiff), revision eefd2f7.
-The following files/folders have been removed:
+A copy of [autodiff](https://github.com/autodiff/autodiff), revision 430a0f3.
```

Comment:
Is this a specific commit or the tagged version `v0.5.11`. If it is the later, I would prefer to mention the tagged version here.